rqd/rqd/rqmachine.py

```diff
@@ -45,8 +45,13 @@
 # pylint: disable=import-error,wrong-import-position
 if platform.system() in ('Linux', 'Darwin'):
     import resource
-elif platform.system() == "win32":
-    import win32api
+elif platform.system() == "Windows":
+    winpsIsAvailable = False
+    try:
+        import winps
```

Comment:
Do users need to perform any steps to install `winps`? Or it's just loaded directly from the source directory?

Reply:
Yes, users who are using source code need to install it using `setup.py` if they want to use the module on Windows.

    cd rqd/winps
    python setup.py install

If OpenCue supports PyPI, it can be installed automatically with RQD dependencies.